test/support/hugo_helpers.rb

```
@@ -0,0 +1,61 @@
+      use RequestLogger if ENV["DEBUG"]
+      use Rack::Static,
+        urls: [""],
+        root: that.destination_path,
+        index: "index.html",
+        header_rules: [
+          # Cache all static files in public caches (e.g. Rack::Cache)
+          #  as well as in the browser
+          [:all, {"cache-control" => "public, max-age=31536000"}],
+          # Provide web fonts with cross-origin access-control-headers
+          #  Firefox requires this when serving assets using a Content Delivery Network
+          [:fonts, {"access-control-allow-origin" => "*"}]
```

rubocop

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)